client/client.go

```diff
@@ -187,11 +190,24 @@ func loadServerTrust(conf Config) (*x509.CertPool, error) {
 	// Read certificate file.
 	servcert, err := ioutil.ReadFile(conf.ServerCert)
 	if err != nil {
+		log.Error("/etc/mender/server.crt is inaccessible.")
 		return nil, err
 	}
 
 	if len(servcert) == 0 {
-		return nil, errors.New("unable to find system and server certificates")
+		log.Error("/etc/mender/server.crt is empty.")
+		return nil, errors.New("server certificate is empty")
+	}
+
+	block, _ := pem.Decode([]byte(servcert))
+	if block != nil {
+		cert, err := x509.ParseCertificate(block.Bytes)
+		if err == nil {
+			log.Info("Have correct server certificate.")
```

Comment:
What do you mean with "Have correct server certificate"? Do we know that for sure? Maybe drop this and print only the other two log messages.

Reply:
I can drop this, but what I meant was that the certificate is there and is formatted correctly, since we can parse it. I will make the message more explicit.

Reply:
I think it can very easily be misunderstood to mean: "The certificate matches the one from the server", but this is wrong. All we know is that it parsed correctly. Maybe instead write "certificate correctly parsed"?